### DISCUSSION TITLE
Move planning-related command language code into this repository

### DISCUSSION THREAD

**Discussion by marip8** [tesseract-robotics/tesseract_planning#7] [Ideas]
A majority of the command language code seems to be implementations of the `Instruction` class related to planning tasks (waypoints, move instructions, etc.). Code in the main tesseract repository probably doesn't (or maybe shouldn't) need to know about planning specific command language implementations. I think we should move that code into this repository to improve overall code cohesiveness and enable easier changes to the command language code (i.e. no "dual" pull requests against this and the main repository)

**Comment by Levi-Armstrong** [comment]
I agree with you on this, which is what we originally did but then realized that tesseract_visualization depended on it. One option would be to remove the dependency of the tesseract_visualization on the command language to solve the issues.

I would like to change the visualization package such that we define a set of markers types which are used. Then user must convert to these supported marker types. Then we would create utilities in Tesseract Planning to convert the command language to the correct types. What do you think?

Marker Types
- Geometry Marker: Tesseract Geometry Types
- Arrow
- Line
- Line Strips
- Axis
- Toolpath
- Trajectory
- Scene

Additional functionality which is not a marker
- EnvState
